test/matchers/toEqualParsedQuery.js

```diff
@@ -66,6 +67,15 @@ let objectsEqual = function (received, expected){
     }
 };
 
+function isTerm(value) {
+    return value instanceof DefaultGraph
```

Comment:
Probably doesn't matter too much since this is just used for unit testing, but something like `'termType' in value` might be simpler.

Reply:
I tried that but using something like `return 'termType' in value` leads to a significant number of errors similar to `other.datatype.equals is not a function`. Adding more checks started to look really ugly, so I opted to say with multiple `instanceof`s.